### DISCUSSION TITLE
Wrong ascendant hence wrong vedic kundali

### DISCUSSION THREAD

**Discussion by knowledgeira** [g-battaglia/kerykeion#75] [General]
I have inserted proper birth details values of me and my friends...the op always produces wrong +1 ascendant house.
For example i have saggi ascendant .. your software produces capricon as ascendant and thus produces all wrong houses and signs. 
>>> from kerykeion import Report, KrInstance
>>> manoj = KrInstance("manoj je", 1986, 6, 24, 19, 40, "baramati" , "India")
>>> report = Report(manoj)
>>> report.print_report()

+- Kerykeion report for manoj jagdale -+
+-----------+-------+--------------+-----------+----------+
| Date      | Time  | Location     | Longitude | Latitude |
+-----------+-------+--------------+-----------+----------+
| 24/6/1986 | 19:40 | baramati, IN | 74.57767  | 18.15174 |
+-----------+-------+--------------+-----------+----------+
+-----------+------+-------+------+----------------+
| Planet    | Sign | Pos.  | Ret. | House          |
+-----------+------+-------+------+----------------+
| Sun       | Can  | 2.77  | -    | Sixth House    |
| Moon      | Aqu  | 6.92  | -    | First House    |
| Mercury   | Can  | 28.06 | -    | Seventh House  |
| Venus     | Leo  | 10.31 | -    | Seventh House  |
| Mars      | Cap  | 21.61 | R    | First House    |
| Jupiter   | Pis  | 22.33 | -    | Third House    |
| Saturn    | Sag  | 4.49  | R    | Eleventh House |
| Uranus    | Sag  | 19.85 | R    | Twelfth House  |
| Neptune   | Cap  | 4.48  | R    | Twelfth House  |
| Pluto     | Sco  | 4.66  | R    | Tenth House    |
| Mean_Node | Ari  | 26.58 | R    | Fourth House   |
| True_Node | Ari  | 27.64 | R    | Fourth House   |
+-----------+------+-------+------+----------------+
+----------------+------+----------+
| House          | Sign | Position |
+----------------+------+----------+
| First House    | Cap  | 10.58    |
| Second House   | Aqu  | 13.25    |
| Third House    | Pis  | 18.03    |
| Fourth House   | Ari  | 21.12    |
| Fifth House    | Tau  | 19.95    |
| Sixth House    | Gem  | 15.51    |
| Seventh House  | Can  | 10.58    |
| Eighth House   | Leo  | 13.25    |
| Ninth House    | Vir  | 18.03    |
| Tenth House    | Lib  | 21.12    |
| Eleventh House | Sco  | 19.95    |
| Twelfth House  | Sag  | 15.51    |
+----------------+------+----------+

**Comment by g-battaglia** [comment]
Mmm, the result of the Kerykeion library looks good. For comparison here the same with Astro Dienst:
https://www.astro.com/cgi/chart.cgi?;nhor=1;cid=84vfileyQvbfy-u1690974155

**Comment by knowledgeira** [comment]
Myself small astrologer ..t my ascendant is saggi...by vedic panchang method (offline) ...for online you can check any indian website/app..

For example : https://www.astrosage.com/kundli/
all the planets positions also changed by + 1
